src/Scroll/ScrollView.tsx

```diff
@@ -119,18 +127,19 @@ export default class ScrollView extends React.Component {
     };
   };
 
-  scrollTo = (_x, _y, scrollDuration) => {
+  scrollTo = (_x: number, _y: number, scrollDuration: number) => {
     const { scrollTop, scrollLeft } = this.getScrollView();
     const x = _x === null ? scrollLeft : _x;
     const y = _y === null ? scrollTop : _y;
```

Comment:
`_` cannot be `null` (given its type) so this is a fruitless check. Should it have been `number | null`?